client/components/infinite-scroll/index.js

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { defer, throttle } from 'lodash';
+
+const SCROLL_CHECK_RATE_IN_MS = 400;
+
+export default class InfiniteScroll extends Component {
+	static propTypes = {
+		nextPageMethod: PropTypes.func.isRequired,
+	};
+
+	checkScrollPositionHandler = throttle(
+		this.checkScrollPosition.bind( this, true ),
```

Comment:
nowadays I'm more used to seeing an auto-bound arrow function `() => this.checkScrollPosition( true )` than a bind.

Reply:
On the other hand, I heard that `bind`-ing is significantly faster than creating a new arrow function instance. Not that it would matter in this case at all (who cares about one more object allocated when rendering a list? The React elements created by JSX will always totally dominate). But it kind of swayed me towards leaving the status-quo as is 🙂 I also usually prefer arrow functions, as I find them more readable than `bind` or `_.partial`.

Reply:
Pushed a commit that changes this to arrow function.